docs/docs/references/connectors/README.mdx

```diff
@@ -0,0 +1,114 @@
+import ConnectorPlatformIntro from './fragments/_connector_platform_intro.mdx';
+import ConnectorStorageIntro from './fragments/_connector_storage_intro.mdx';
+import ConnectorTypeIntro from './fragments/_connector_type_intro.mdx';
+
+# 🔌 Connectors
+
+## Introduction
+
+### What is a connector?
+
+_Connectors_ play a critical role in Logto. With their help, Logto enables end-users to use passwordless registration or sign-in and the capabilities of signing in with social accounts. With the increasing popularity of websites and applications, passwordless and social sign-ins allow users to avoid managing numerous accounts and passwords.
+
+:hushed: Sounds fascinating, uh? See "[Configure connectors](../../recipes/configure-connectors/)" if you want to set up an existing connector. If you can not find the connector you want to set up, you may develop those connectors by following the guides in "[Create your connector](../../recipes/create-your-connector/)".
+
+## Compositions
+
+As there are lots of properties in connector data, and to make the data loading and updating more efficient, we store part of connector data which will be modified frequently to DB and the rest of that locally.
```

Comment:
Hard to read. I think so as well.

<img width="1324" alt="image" src="https://user-images.githubusercontent.com/10594507/176881244-bb033b08-f3d7-47e1-9fab-b889275509c5.png">

just suggestion:

```suggestion
There are lots of properties in connector data.
To make the connector data loading and updating more efficient, 
we store the frequently updated data in DB and the rest in the locality.
```

Reply:
Updated, good call.